ui/src/components/MultiPanelTabs.vue

```diff
@@ -1,7 +1,20 @@
 <template>
     <Splitpanes class="default-theme" @resize="onResize">
-        <Pane v-for="(panel, panelIndex) in panels" min-size="10" :key="panelIndex" :size="panel.size">
+        <Pane 
+            v-for="(panel, panelIndex) in panels" 
+            min-size="10" 
+            :key="panelIndex" 
+            :size="panel.size"
+            draggable="true"
+            @dragstart="(e) => panelDragStart(e, panelIndex)"
+            @dragover.prevent="(e) => panelDragOver(e, panelIndex)"
+            @dragleave.prevent="panelDragLeave"
+            @drop.prevent="(e) => panelDrop(e, panelIndex)"
+            :class="{'panel-dragover': panel.dragover}"
+        >
             <div class="editor-tabs-container">
+                <el-button :icon="DragVertical" link class="tab-icon" />
```

Comment:
Let's make this a drag handle - meaning - user can move entire panel only by interacting with it, not the entire pane as it is now.